packages/node_modules/@node-red/editor-client/src/js/keymap.json

```diff
@@ -51,6 +52,7 @@
         "shift-down": "core:step-selection-down",
         "shift-left": "core:step-selection-left",
         "ctrl-shift-j": "core:show-previous-tab",
-        "ctrl-shift-k": "core:show-next-tab"
+        "ctrl-shift-k": "core:show-next-tab",
+        "ctrl-shift-z": "core:redo"
```

Comment:
This shortcut does not need to be added twice. You've already added it on line 34 to match the `core:undo` action - don't add it here as well

Reply:
`core:undo` is also added twice at line 33 and line 43, and if remove this line redo doesn't work.
Should I move this line to line 44 for clarification?

Reply:
Sorry, I scanned through to check if `undo` was listed twice and didn't spot it!

> Should I move this line to line 44 for clarification?
Yes, that would be helpful.

Reply:
> > Should I move this line to line 44 for clarification?

> Yes, that would be helpful.

I fixed it in a recent commit.